MPCHomeControl/src/model.rs

```diff
@@ -130,6 +166,12 @@ pub struct Boundary {
     pub area: Area,
 }
 
+impl Boundary {
+    pub fn convection_conductance(&self) -> ThermalConductance {
+        todo!("Figure out how to model this!")
```

Comment:
Hlavni problem teto verze. Vubec netusim jak se to pocita, jeste stale jsem nevidel ty videa .... :-( Pokud mas nejaky insight, tak sem s nim. Taky si nejsem jisty jestli (kdyz uz tuto hodnotu spocitam) ji pak spravne pouziju.